mono/sgen/sgen-fin-weak-hash.c

```diff
@@ -451,7 +451,7 @@ add_stage_entry (int num_entries, volatile gint32 *next_entry, StageEntry *entri
 				 * This seems like a good value.  Determined by timing
 				 * sgen-weakref-stress.exe.
 				 */
-				g_usleep (200);
+				mono_thread_info_usleep (200);
```

Comment:
This would trigger https://jenkins.mono-project.com/job/test-mono-mainline-coop/label=debian-amd64/8/parsed_console/log_content.html#WARNING1